File: fe_flutter_ui/lib/models/address.dart
class Addresses {
  int? id;
  String? userId;
  String? addrressValue;
  String? nameAddress;
  String? description;
  bool? status;

  Addresses(
      {this.id,
      this.userId,
      this.addrressValue,
      this.nameAddress,
      this.description,
      this.status});

  Addresses.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    userId = json['userId'];
    addrressValue = json['addrressValue'];
    nameAddress = json['nameAddress'];
    description = json['description'];
    status = json['status'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['id'] = id;
    data['userId'] = userId;
    data['addrressValue'] = addrressValue;
    data['nameAddress'] = nameAddress;
    data['description'] = description;
    data['status'] = status;
    return data;
  }
}

File: be_aspnet_api/MrSoaiAPI/Model/Address.cs
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace MrSoaiAPI.Model
{
    public class Address
    {

        public int Id { get; set; }
        public string UserId { get; set; }
        public MrSoaiUser User { get; set; }
        public string AddrressValue { get; set; }
        public string NameAddress { get; set; }
        public string Description { get; set; }
        [DefaultValue(true)]
        public bool Status { get; set; } = true;
    }
}
